Corrections in UMD and CMU faculty

saptab opened the pull request:
I have corrected faculty details in UMD. I have added faculty details in UMD. However, there is a strange issue that the records of Prof. Mihai Pop, Prof. Alan Sussman and Prof. Howard C. Elman are present in the CSV but are not showing up in the ranking. They have published in the last decade. Their DBLP names are correct and Prof. Pop and Prof. Sussman are indexed in articles.json file. I also  moved Prof. Jaime Carbonell in CMU to old/rip.csv after his death in February,2020.

The corrections are correctly shown in the file on which diff can be done. Thank you so much.

# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

_Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

**Inclusion criteria**

- [ ] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

saptab committed:
I have corrected faculty details in UMD. I have added faculty details in UMD. However, there is a strange issue that the records of Prof. Mihai Pop, Prof. Alan Sussman and Prof. Howard C. Elman are present in the CSV but are not showing up in the ranking. They have published in the last decade. Their DBLP names are correct and Prof. Pop and Prof. Sussman are indexed in articles.json file. I also  moved Prof. Jaime Carbonell in CMU to old/rip.csv after his death in February,2020.

saptab commented:
@emeryberger Kindly consider if the commit can be merged. I have done all the necessary steps as advised by you in #2633, last week which is explained in the comments above.

emeryberger commented on csrankings.csv:
```diff
@@ -8956,8 +8964,8 @@ Jon Barker,University of Sheffield,https://www.sheffield.ac.uk/dcs/people/academ
 Jon Doyle,North Carolina State University,https://www.csc.ncsu.edu/faculty/doyle,C6DtGmMAAAAJ
-Jon E. Froehlich,University of Washington,http://www.cs.umd.edu/~jonf,nExKrpsAAAAJ
-Jon Froehlich,University of Washington,http://www.cs.umd.edu/~jonf,nExKrpsAAAAJ
+Jon E. Froehlich,University of Maryland - College Park,http://www.cs.umd.edu/~jonf,nExKrpsAAAAJ
```
I spot checked this listing and Froehlich's web page says he has been at UW since 2017.

saptab replied:
You are correct. I missed the recent affiliation of Dr. Froehlich. I have corrected it to University of Washington in the csrankings.csv file.

emeryberger commented on csrankings.csv:
```diff
@@ -8982,8 +8990,8 @@ Jonathan E. Rowe,University of Birmingham,https://www.birmingham.ac.uk/staff/pro
-Jonathan K. Lazar,Towson University,http://orion.towson.edu/~jlazar,CDpfbBsAAAAJ
-Jonathan Katz,George Mason University,https://www.cs.umd.edu/~jkatz,H4rKFc8AAAAJ
+Jonathan K. Lazar,University of Maryland - College Park,https://ischool.umd.edu/faculty-staff/jonathan-lazar,CDpfbBsAAAAJ
+Jonathan Katz,University of Maryland - College Park,https://www.cs.umd.edu/~jkatz,H4rKFc8AAAAJ
```
Web page clearly states no longer at Maryland.

saptab replied:
You are correct. I missed the recent affiliation of Prof. Jonathan Katz. I have corrected it to George Mason University in the csrankings.csv file.

emeryberger commented on csrankings.csv:
```diff
@@ -17640,7 +17656,7 @@ Søren Hauberg,DTU,https://www.compute.dtu.dk/english,NOSCHOLARPAGE
 Søren Kimmer Schou Gregersen,DTU,https://www.compute.dtu.dk/english,NOSCHOLARPAGE
 Søren Lauesen,IT University of Copenhagen,https://pure.itu.dk/portal/en/persons/soeren-lauesen(ea6d10cf-6bb5-4414-8966-f1ed2591f794).html,NOSCHOLARPAGE
-Süleyman Cenk Sahinalp,Indiana University,https://www.soic.indiana.edu/all-people/profile.html?profile_id=291,NOSCHOLARPAGE
+Süleyman Cenk Sahinalp,University of Maryland - College Park,https://www.cs.umd.edu/people/sahinalp,O8wbTncAAAAJ
```
Apparently not at Maryland either. https://ccr.cancer.gov/cancer-data-science-laboratory/s-cenk-sahinalp

saptab replied:
You are correct. I missed the recent affiliation of Prof. Sahinalp. I have corrected it to National Institute of Health in the csrankings.csv file.

emeryberger commented on csrankings.csv:
```diff
@@ -7381,7 +7387,7 @@ Ian M. Mitchell,University of British Columbia,https://www.cs.ubc.ca/~mitchell,b
 Ian McKillop,University of Waterloo,https://uwaterloo.ca/public-health-and-health-systems/people-profiles/ian-mckillop,NOSCHOLARPAGE
 Ian McQuillan,University of Saskatchewan,https://www.cs.usask.ca/faculty/mcquillan,NOSCHOLARPAGE
-Ian Miguel,University of St Andrews,https://ianm.host.cs.st-andrews.ac.uk,IS6YSBkAAAAJ
+Ian Miers,University of Maryland - College Park,https://www.cs.umd.edu/people/imiers,21kN_WsAAAAJ
```
Visting prof, not eligible for inclusion.

saptab replied:
I had included Dr. Miers as a newly joined Assistant Professor at the University of Maryland, as per their news article from 2019 https://www.cs.umd.edu/article/2019/06/5-new-professors-join-computer-science-faculty (It says that Dr. Miers is joining in the time period of July 1-15, 2020. Kindly advise what should be done with this data entry.

emeryberger replied:
That's fine, thanks.

emeryberger reviewed (changes requested):
Practically every institution change I checked is wrong. Please fix the ones I highlighted and check all.

saptab commented:
I have reviewed all the modifications.

The removal of the CMU entry is correct as Dr. Carbonell passed away in early 2020. His obituary is https://www.cmu.edu/news/stories/archives/2020/february/jaime-carbonell-obituary.html.

Among the 4 mistakes regarding UMD affiliation that you have pointed out, I have corrected 3 data entries of Dr. Jon Froehlich to the University of Washington, Dr. Katz to George Mason University and 
Dr. Cenk Sahinalp. to the National Institute of Health.

I had included Dr. Miers as a newly joined Assistant Professor at the University of Maryland, as per their news article from 2019 https://www.cs.umd.edu/article/2019/06/5-new-professors-join-computer-science-faculty (It says that Dr. Miers is joining in the time period of July 1-15, 2020. Kindly advise what should be done with this data entry.

I have reviewed the remaining affiliations and removed Dr. Stephen Altschul's entry as he seems to be also affiliated to the National Institute of Health.

The other entries appear to all be affiliated at the University of Maryland, right now. 

Please let me know if I can do anything else. I would be happy to do so. :D

I am pushing the corrections to github once again for your kind perusal.

saptab committed:
Please consider the corrections, addressing your feedback. I have also provided the detailed updates in a github comment.

emeryberger commented on csrankings.csv:
```diff
@@ -17640,7 +17655,7 @@ Søren Hauberg,DTU,https://www.compute.dtu.dk/english,NOSCHOLARPAGE
 Søren Kimmer Schou Gregersen,DTU,https://www.compute.dtu.dk/english,NOSCHOLARPAGE
 Søren Lauesen,IT University of Copenhagen,https://pure.itu.dk/portal/en/persons/soeren-lauesen(ea6d10cf-6bb5-4414-8966-f1ed2591f794).html,NOSCHOLARPAGE
-Süleyman Cenk Sahinalp,Indiana University,https://www.soic.indiana.edu/all-people/profile.html?profile_id=291,NOSCHOLARPAGE
+Süleyman Cenk Sahinalp,National Institute of Health,https://www.cs.umd.edu/people/sahinalp,O8wbTncAAAAJ
```
This entry should just be removed.

saptab replied:
I have removed the data entry of Cenk Sahinalp. Please merge.

emeryberger reviewed (changes requested):
Please delete the entry for Sahinalp and I will merge.

saptab committed:
I have removed the data entry of Cenk Sahinalp. Please merge.
